data/core/faqs.md

```diff
@@ -7,6 +7,37 @@
     - respond_faq
     - action_set_faq_slot
 
+## faqs
+* faq{"product": "x"}
+    - slot{"product": "x"}
+    - respond_faq
+    - action_set_faq_slot
+
+## faqs
+* faq{"product": "rasa"}
+    - slot{"product": "rasa"}
+    - respond_faq
+    - action_set_faq_slot
+
+## faqs
+* faq{"product": "nlu"}
+    - slot{"product": "nlu"}
+    - respond_faq
+    - action_set_faq_slot
+
+## faqs
+* faq{"product": "core"}
+    - slot{"product": "core"}
+    - respond_faq
+    - action_set_faq_slot
+
+## faqs
+* faq{"product": "all"}
+    - slot{"product": "all"}
+    - respond_faq
+    - action_set_faq_slot
+
```

Comment:
Re: Your comment about `faq: use_entities: []` -- I _think_ we can actually still have that as long as we have stories where the slot event happens? Because i think use entities only unfeaturizes the entity -- if the categorical slot auto-fills then we can still split on that. wdyt?

Reply:
So, I thought that this is what happens, but experimentation suggested it wasn't. I'd like to test it out again though because I might've missed something (and I can't remember exactly what I tried now of course =P)

Reply:
you mean using this, with `use_entities=[]` for faqs should work right?
```
## faqs
* faq
    - slot{"product": "x"}
    - respond_faq
    - action_set_faq_slot

## new to rasa at start
* how_to_get_started{"user_type": "new"}
    - action_set_onboarding
    - slot{"onboarding": true}
    - utter_getstarted_new
    - utter_built_bot_before
* deny
    - utter_explain_rasa_components
    - utter_rasa_components_details
    - utter_ask_explain_nlucorex
* faq
    - slot{"product":"rasa"}
    - utter_explain_nlu
    - utter_explain_core
```

Reply:
Ok, I just tested it again, when `use_entities=[]` then the slot is not autofilled. Is this behaviour documented?